test/sequential/sequential.status

```diff
@@ -10,6 +10,7 @@ prefix sequential
 test-inspector-async-call-stack       : PASS, FLAKY
 test-inspector-bindings               : PASS, FLAKY
 test-inspector-debug-end              : PASS, FLAKY
+test-inspector-async-hook-setup-at-signal:  PASS, FLAKY
```

Comment:
Hm, is this related to the change here?

Reply:
No, this snuck its way in here because `v8.x-staging` was rebased.

The commits should still apply cleanly; I can rebase this PR if @gibfahn likes.